content/tracing/languages/java.md

```diff
@@ -224,6 +225,33 @@ Now add `@Trace` to methods to have them be traced when running with `dd-java-ag
 Datadog's Java Tracer provides support for 'in-process' JMX metrics collection. This is enabled with `jmxfetch.enabled` configuration parameter. Additional JMX metrics are configured using configuration files that are passed to `jmxfetch.metrics-configs`. Contents of those configuration files are equivalent to contents of the `conf` section for external jmxfetch. See [JMX Integration][8] for further details on configuration.
 By default, when JMX metrics collection is enabled it monitors JVM heap memory, thread count, and garbage collection. Use it in conjunction with APM for a broader view into your Java application's performance.
 
+## Logs Slf4j+MDC Autoinjection
```

Comment:
Those instructions are a duplicate from https://docs.datadoghq.com/tracing/advanced_usage/?tab=java#logging.
Having the same content in different place is rarely a good practice.

I would suggest to just add a link to the logging java section so if one day we need to make a change or an update, we don't need to change 3 different parts of the documentation.

So the documentation that actually should be updated is https://docs.datadoghq.com/tracing/advanced_usage/?tab=java#logging with the flag you have there.

Reply:
Sure I'm all for reduced duplication. I would like to keep the flag in the config section because customers expect to find all tracer configurations there. I added a link to the advanced logging section from there.

Also, I found that `advanced_usage` section confusing so I updated the wording and format a bit. Please let me know what you think.